### PR TITLE
Add freja and bankid icons

### PR DESCRIPTION
**Pull request by robsan-nordnet**

### CODE REVIEW

**Review by bmd007 — APPROVED** [review]

**Review by brutisso — APPROVED** [review]

**Review by jesperwiner — CHANGES_REQUESTED** [review]
optimize the SVGs and fix currentColor please

**Review by brutisso — DISMISSED** [review]

File: src/Atoms/Icon/finance/freja/Freja24.tsx [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+
+import { IconBase } from '../../IconBase';
+import { IconProps } from '../../IconBase.types';
+
+const Freja16: React.FC<IconProps> = (props) => {
+    return (
+        <IconBase {...props} width={16} height={16}>
+            <svg width="16" height="16" fill="none">
```
Comment (jesperwiner) [inline src/Atoms/Icon/finance/freja/Freja24.tsx:9]: should be 24 on all these right ?

File: src/Atoms/Icon/finance/bankId/index.ts [hunk 2]
```diff
@@ -1,5 +1,9 @@
+import BankId16 from './BankId16';
 import BankId24 from './BankId24';
+import BankId32 from './BankId32';
 
 export default {
   BankId24,
+  BankId16,
+  BankId32,
```
Comment (jesperwiner) [inline src/Atoms/Icon/finance/bankId/index.ts:8]: if you have OCD these can be sorted :)

File: src/Atoms/Icon/finance/freja/Freja24.tsx (hunk 1 above)
Comment (brutisso) [reply src/Atoms/Icon/finance/freja/Freja24.tsx]: Nice catch! Yes, definitely 24. Fixed that now.

File: src/Atoms/Icon/finance/bankId/index.ts (hunk 2 above)
Comment (brutisso) [reply src/Atoms/Icon/finance/bankId/index.ts:8]: Fixed :)

**Review by jesperwiner — APPROVED** [review]

**Pull request merged**